bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -1807,42 +1935,53 @@ paths:
               $ref: '#/components/headers/API-Version'
           content:
             application/json:
-              schema:
-                $ref: '#/components/schemas/BookingRefCancelledStatus'
               examples:
                 bookingCancelledExample:
                   summary: |
                     Entire Booking cancelled
                   description: |
-                    The consumer has requested that the entire `Booking` should be cancelled. `bookingStatus` is set to `CANCELLED` and `amendedBookingStatus` is set to `AMENDMENT CANCELLED`.
-                  value:
-                    carrierBookingRequestReference: cbrr-123
-                    carrierBookingReference: cbr-987
-                    bookingStatus: CANCELLED
-                    amendedBookingStatus: AMENDMENT CANCELLED
-                amendmentCancelled1Example:
+                    The consumer has requested that the entire `Booking` should be cancelled. The cancellation has been accepted and is now awaiting further processing by the provider
+                amendmentCancelledExample:
                   summary: |
                     Amendment cancelled on a confirmed Booking
                   description: |
-                    The consumer has requested that the amendment to a confirmed `Booking` is to be cancelled. The status of the `Booking` stays as `CONFIRMED`. The `amendedBookingStatus` is set to `AMENDMENT CANCELLED`.
-                  value:
-                    carrierBookingRequestReference: cbrr-123
-                    carrierBookingReference: cbr-987
-                    bookingStatus: CONFIRMED
-                    amendedBookingStatus: AMENDMENT CANCELLED
-                amendmentCancelled2Example:
+                    The consumer has requested that the amendment to a confirmed `Booking` is to be cancelled. The cancellation has been accepted and is now awaiting further processing by the provider
+        '400':
+          description: |
+            In case the Cancel payload does not schema validate a `400` (Bad Request) is returned
```

Comment:
Do we need to Capitalize 'C' in Cancel here?

Reply:
changed to small :-)